fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/transactionprocessor/AbstractLoanRepaymentScheduleTransactionProcessor.java

```diff
@@ -204,12 +204,36 @@ public ChangedTransactionDetail reprocessLoanTransactions(final LocalDate disbur
                 reprocessChargebackTransactionRelation(changedTransactionDetail, transactionsToBeProcessed);
             } else if (loanTransaction.isChargeOff()) {
                 recalculateChargeOffTransaction(changedTransactionDetail, loanTransaction, currency, installments);
+            } else if (loanTransaction.isAccrualActivity() && loanTransaction.isNotReversed()) {
```

Comment:
`loanTransaction.isNotReversed()` is not needed. We filter earlier out the reverted transactions.

Reply:
I did not spotted, will remove